constant/constants.go

```diff
@@ -117,3 +117,199 @@ var ExpTable = [...]int32{15, 34, 57, 92, 135, 372, 560, 840, 1242, 1144, // Beg
 	894634784, 943660770, 995373379, 1049919840, 1107455447, 1168144006, 1232158297, 1299680571,
 	1370903066, 1446028554, 1525246918, 1608855764, 1697021059, // 0 is the amount of exp needed for level 200 to level up i.e. never shall
 }
+
+const (
```

Comment:
If we made these constants of type Skill we could write a `String() string` method that does a map lookup to return the human readable skill name for debugging and logging.